src/gnomescreencast.c

```diff
@@ -362,23 +443,30 @@ gnome_screen_cast_session_record_selections (GnomeScreenCastSession *gnome_scree
     {
       ScreenCastSourceType source_type;
       g_autofree char *key = NULL;
+      GVariant *variant;
```

Comment:
IIRC you must free this. I.e. make it `g_autoptr(GVariant) variant = NULL;`

Reply:
Nice. Done!